lib/aur-repo

```diff
@@ -3,16 +3,22 @@
 readonly argv0=repo
 readonly PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 readonly AUR_COLOR=${AUR_COLOR-auto}
+readonly AURVCS=${AURVCS:-^.*-(cvs|svn|git|hg|bzr|darcs)$}
```

Comment:
I would argue we don't need this for the simple reason that the user can do more complicated things already, i.e.
```
aur repo --list | pcregrep <some_morbid_perl_regex>
```

Reply:
With "this", do you mean the PR in general, or allowing to override `$AURVCS`?

We're already storing that for `--devel` use, allowing to override this has no extra maintenance.

But using the same logic, why implement `--devel` at all? We can just recommend

`aur repo --list | grep -P '^.*-(cvs|svn|git|hg|bzr|darcs)$'` on `aurvcs(7)`. And when the RPC adds proper support for marking packages as devel, we add `--devel`.

I'm fine with either.

Reply:
I just meant AURVCS in this case. I guess the "warning" on VCS packages is about the closest we'll ever get to proper support for marking packages as devel.